peripherals/sdio/sdio_f446.yaml

```diff
@@ -0,0 +1,258 @@
+_modify:
+  SDMMC:   # renaming block to be RM0390-compliant
+    name: SDIO
+    description: 'Secure digital input/output SD/SDIO MMC card host interface'
+    groupName: SDIO
+    addressBlock:
+      size: "0x400"     
+
+SDIO:
+  _strip: SDMMC_
+  _delete:
+    - '*ACKTIMER'
+    - '*IDMA*'
+    - '*VER'
+    - '*ID'
+  _modify:
+    SDMMC_POWER:
+      description: 'SDIO power control register'
+    SDMMC_CLKCR:
+      description: 'CLKCR register controls the SDIO_CK output clock.'
+    SDMMC_CMDR:
+      name: "SDMMC_CMD"  # NOTE: something weird happens on leaving just 'CMD' -- 'MD' -- ?
```

Comment:
Not sure what you mean here exactly.. is something weird still happening with the patch as-is?

Reply:
I'm not sure what exactly am I doing wrong (or is this just as designed, and I've missed that part of svdtools documentation), but the `_strip: SDMMC_` directive strips the first letter of any register starting with C; e.g., if this line 22 is `name: "CMD"`, the register name becomes "MD" (and the patch fails).